### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**

### CODE REVIEW

File: vignettes/DGE.ToolsPlotGallery.Rmd [hunk 1]
```diff
@@ -308,13 +330,15 @@ printAndSave(BCV_Plot, file.path(outputPath, "BCV_Plot.png"))
...
 This function plots distributions before and after normalization as a two panel faceted plot.  It uses `edgeR::calcNormFactors()` and thus supports any normalization accepted by `calcNormFactors()`.  
 
-Input can be a DGEobj or a raw counts matrix.  Output can be a box plot or density distribution.  
+Input can be a DGEobj or a raw counts matrix.  Output can be a canvasXpress or ggplot object that is either a box plot or density distribution. 
+
+Default plot type is a canvasXpress and default plot category is box plot.
 
 ```
-boxNorm <- plotNorm(dgeObj, plotType = "box", normalize = "tmm")
+boxNorm <- plotNorm(dgeObj, plotCategory = "box", normalize = "tmm")
```
Comment (saranya-ag) [inline vignettes/DGE.ToolsPlotGallery.Rmd]: You need to add the plotType as well in the Vignette

File: R/plotNorm.R [hunk 2]
```diff
@@ -1,100 +1,204 @@
...
 #'   normalization.
 #'
-#' @return A faceted ggplot plot showing before/after log2CPM normalization.
+#' @return A faceted canvasXpress/ggplot plot showing before/after log2CPM normalization.
 #'
 #' @examples
 #' \dontrun{
-#'    myNormPlotBox <- plotNorm(myDGEobj, plotType = "box")
-#'    myNormPlotDensity <- plotNorm(counts, plotType = "density")
+#'    myNormPlotBox <- plotNorm(myDGEobj, plotCategory = "box")
```
Comment (saranya-ag) [inline R/plotNorm.R]: you need to add plotType here.

**Review by saranya-ag — CHANGES_REQUESTED** [review]

File: R/plotNorm.R (hunk 2 above)
Comment (agenius-mohammed-ali) [reply R/plotNorm.R]: done

File: vignettes/DGE.ToolsPlotGallery.Rmd (hunk 1 above)
Comment (agenius-mohammed-ali) [reply vignettes/DGE.ToolsPlotGallery.Rmd]: done

**Review by saranya-ag — APPROVED** [review]

**Pull request merged**